package.json

```diff
@@ -123,6 +124,6 @@
   ],
   "license": "MIT",
   "engines": {
-    "node": "^6.14.0 || ^8.10.0 || >=9.10.0"
+    "node": "^8.10.0 || ^10.13.0 || >=11.10.1"
```

Comment:
Can we make this change in a separate PR?

Reply:
Yeah, I noted it in the PR description:

> Changes engines.node to use Object.entries for now (this is the reason that this PR has "don't merge" label. I assume this PR will be merged after the PR which updates engines.node was merged.)

Reply:
Oh, sorry, I missed that. Ok, sounds fine to me.